demo/NPlaylist.ReadingPlaylists/Program.cs

```diff
@@ -1,11 +1,20 @@
 using System;
+using System.IO;
+using NPlaylist.Asx;
 
 namespace NPlaylist.ReadingPlaylists
 {
     internal class Program
     {
         private static void Main(string[] args)
         {
+            string str = File.ReadAllText(@"C:\list.asx");
```

Comment:
Be more verbose with variable names
```suggestion
            string fileContent = File.ReadAllText(@"C:\list.asx");
            var asxDeserializer= new AsxDeserializer();
            var playlist = t.Deserialize(fileContent);

            var asxSerializer = new AsxSerializer();
            Console.WriteLine(asxSerializer.Serialize(playlist));
```

Reply:
I forgot to delete this code from main program, it should be empty